### PR TITLE
feat: add Japanese support and update translation

### LINKED ISSUE #2364: The upgrade prompt of AFFiNE Cloud has incomplete translation
image
I have switched to Chinese.

### LINKED ISSUE #2368: Spacing needs to be added between Chinese and English characters when typing in mixed Chinese-English
image
Currently, spacing is added in some cases but not in others.

### LINKED ISSUE #2374: The button text for switching themes needs to have the first letter capitalized
image

### PR DESCRIPTION
**Pull request by JimmFly**
Close #2368  
Close #2374 
Close #2364
af7fb56b-99bf-4a07-8136-a9704ec9f21c

### CODE REVIEW

File: packages/i18n/src/resources/ru.json [hunk 1]
```diff
@@ -9,15 +9,19 @@
...
   "is a Local Workspace": "это локальное рабочее пространство",
   "Wait for Sync": "Дождитесь синхронизации",
   "Joined Workspace": "Присоединенное рабочее пространство",
   "Retain local cached data": "Сохранять локальные кэшированные данные",
   "will delete member": "удалит участника",
   "Download all data": "Скачать все данные",
-  "It takes up little space on your device": "Занимает мало места на вашем устройстве.",
-  "It takes up more space on your device": "Занимает много места на вашем устройстве.",
+  "It takes up little space on your device": {
+    "": "Занимает мало места на вашем устройстве."
```
Comment (pengx17) [inline packages/i18n/src/resources/ru.json:20]: ☝️

File: packages/i18n/src/resources/ru.json [hunk 2]
```diff
@@ -9,15 +9,19 @@
...
+  },
+  "It takes up more space on your device": {
```
Comment (pengx17) [inline packages/i18n/src/resources/ru.json:22]: 👇

File: packages/i18n/src/resources/ru.json [hunk 3]
```diff
@@ -120,7 +126,9 @@
   "Delete Workspace placeholder": "Пожалуйста, введите \"Delete\" для подтверждения",
   "Delete Workspace Description": "Удаление (<1>{{workspace}}</1>) нельзя отменить, пожалуйста, действуйте с осторожностью. Все содержимое будет потеряно.",
   "Delete Workspace Description2": "Удаление (<1>{{workspace}}</1>) приведет к удалению как локальных, так и облачных данных, эта операция не может быть отменена, пожалуйста действуйте с осторожностью.",
-  "Once deleted, you can't undo this action": "После удаления вы не сможете отменить это действие.",
+  "Once deleted, you can't undo this action": {
+    "": "После удаления вы не сможете отменить это действие."
```
Comment (pengx17) [inline packages/i18n/src/resources/ru.json:130]: ☝️

File: packages/i18n/src/resources/ru.json (hunk 1 above)
Comment (JimmFly) [reply packages/i18n/src/resources/ru.json:20]: I am currently working on resolving this issue.

**Review by pengx17 — APPROVED** [review]

**Pull request merged**